news/2014-09-30-lumen-learning.html

```diff
@@ -101,9 +101,9 @@ <h3>Lumen Learning</h3>
                     delivery tailored to the needs of disadvantaged students.</p>
                 <p>For more information about the project, read the <a href="http://lumenlearning.com/ann-courseware-challenge/">Lumen Learning press release</a>.</p>
```

Comment:
[http://lumenlearning.com/ann-courseware-challenge/](url) does not work and I could not find a replacement link.